grpc/tests/python/MyGame/Sample/monster_grpc_fb.py

```diff
@@ -0,0 +1,190 @@
+# Generated by the gRPC Python protocol compiler plugin. DO NOT EDIT!
+import grpc
+
+import Monster_fb as Monster__fb
```

Comment:
This gave me an error saying that `Monster_fb` is not a module. Same for `Weapon_fb`.

Reply:
No idea.. you'll know more about Python & gRPC by now than me :)